src/lib/server/Config.cpp

```diff
@@ -30,7 +30,11 @@ const auto kBarrierProtocolOption = "barrier";
 // Config
 //
 
-Config::Config(IEventQueue *events) : m_inputFilter(events), m_hasLockToScreenAction(false), m_events(events)
+Config::Config(IEventQueue *events)
+    : m_inputFilter(events),
+      m_hasLockToScreenAction(false),
+      m_events(events),
+      m_mouseSpeed(1.0)
```

Comment:
Unnecessary, already using default init.

```suggestion
      m_events(events)
```